dvc/output/base.py

```diff
@@ -1,14 +1,16 @@
 import logging
+import os.path
 from urllib.parse import urlparse
 from copy import copy
 
 from voluptuous import Any
 
-import dvc.prompt as prompt
-from dvc.cache import NamedCache
-from dvc.exceptions import CollectCacheError
-from dvc.exceptions import DvcException
-from dvc.remote.base import RemoteBASE
+from .. import prompt
+from ..cache import NamedCache
+from ..exceptions import CollectCacheError
+from ..exceptions import DvcException
+from ..remote.base import RemoteBASE
+from ..utils.fs import path_isin
```

Comment:
I am completely lost here. I would say this is error-prone, `.exceptions` and `..exceptions` will mean different things while being too close looking.

Anyway, relative imports are for packages, which are a thing in itself within a bigger package. Since we have everything global, like global exceptions, global remotes hierarchy, etc we should not use that.

Reply:
@Suor I disagree. That being said, let's remove them from this PR since they are not needed and so we don't argue about them here and right now.

Reply:
Yes like I said https://github.com/iterative/dvc/pull/2975#issuecomment-576038423 should probably leave this for a different PR.